content/millennium/r4/encounters/encounter.md

```diff
@@ -37,11 +37,30 @@ The following fields are returned if valued:
    * [Name](https://hl7.org/fhir/r4/encounter-definitions.html#Encounter.location.location){:target="_blank"}
    * [Location status](https://hl7.org/fhir/r4/encounter-definitions.html#Encounter.location.status){:target="_blank"}
 * [Service provider (Organization)](https://hl7.org/fhir/r4/encounter-definitions.html#Encounter.serviceProvider){:target="_blank"}
+* [Extensions including client organization, estimated financial responsibility amount, payment collection status and estimated financial responsibility not collected reason](#extensions){:target="_blank"}
 
 ## Terminology Bindings
 
 <%= terminology_table(:encounter, :r4) %>
 
+## Extensions
+
+* [Client Organization]
+* [Estimated Financial Responsibility Amount]
+* [Payment Collection Status]
+* [Estimated Financial Responsibility Not Collected Reason]
+
+### Custom Extensions
+
+All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/StructureDefinition/{id}`
+
+ ID                                                       | Value\[x] Type      | Description
+----------------------------------------------------------|---------------------|--------------------------------------------------------------------------
+`client-organization`                                     | [`Reference`]       | The financially responsible organization.
+`estimated-financial-responsibility-amount`               | [`Money`]           | The estimated amount to be collected for the encounter.
+`payment-collection-status`                               | [`CodeableConcept`] | The status of the payment collection for the encounter.
+`estimated-financial-responsibility-not-collected-reason` | [`CodeableConcept`] | The reason no estimated amount is collected for the encounter.
```

Comment:
not sure this is something we want to do, but we could make this line
```
`estimated-financial-responsibility-`<br>`not-collected-reason` | [`CodeableConcept`] | The reason no <br>estimated amount is collected for the encounter.`
```

The table would then look like
<img width="650" alt="Screen Shot 2020-07-01 at 2 38 14 PM" src="https://user-images.githubusercontent.com/56039503/86284474-9233f580-bba8-11ea-8a0e-e874188756e4.png">

Reply:
we're just going to let this table scroll for now